src/Reader/StandaloneExtensionManager.php

```diff
@@ -58,8 +58,10 @@ public function get($extension)
      *
      * @param string $name
      * @param string $class
+     *
+     * @return void
      */
-    public function add($name, $class)
+    public function add($name, $class): void
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.